salt/client/ssh/wrapper/cp.py

```diff
@@ -5,11 +5,11 @@
 # Import Python libs
 from __future__ import absolute_import, print_function
 
-import logging
 import os
 
 # Import salt libs
 import salt.client.ssh
+import salt.log.setup as logging
```

Comment:
Hmm, why are we doing this?

Reply:
My understanding is that, if default Python's logging is used here, it would contribute to the deadlock when this code is executed on the context of bootstrapping the new sub-processes for handling the SSH work.